h2o-py/tests/pyunit_utils/utilsPY.py

```diff
@@ -386,7 +386,7 @@ def javapredict(algo, equality, train, test, x, y, compile_only=False, separator
         out_pojo_csv = os.path.join(tmpdir, "out_pojo.csv")
         cp_sep = ";" if sys.platform == "win32" else ":"
         java_cmd = ["java", "-ea", "-cp", h2o_genmodel_jar + cp_sep + tmpdir, "-Xmx12g", "-XX:MaxPermSize=2g",
-                    "-XX:ReservedCodeCacheSize=256m", "hex.genmodel.tools.PredictCsv",
+                    "-XX:ReservedCodeCacheSize=256m", "hex.genmodel.tools.PredictCsv", "--decimal",
```

Comment:
`--decimal` makes sure to output decimal numbers to the csv (as opposed to hexadecimal).

This fixes `pyunit_javapredict_carsGBM.py` - h2o seems to be unable to parse hexadecimal floats  from the csv and from a brief look in `h2o-core/src/main/java/water/parser/CsvParser.java` seems to not support it at all (if that's really the file that is responsible for csv parsing when using `h2o.upload_file`).